app/wallet/service.js

```diff
@@ -18,26 +14,29 @@ export const getDecodeService = promisify(client.getDecodeMessageService);
 
 export const getNextAddress = (walletService, accountNum) =>
   new Promise((resolve, reject) => {
-    const request = new NextAddressRequest();
+    const request = new api.NextAddressRequest();
     request.setAccount(accountNum);
     request.setKind(0);
+    request.setGapPolicy(api.NextAddressRequest.GapPolicy.GAP_POLICY_WRAP);
```

Comment:
@alexlyp One thing to note about this PR is that for all cases of the client using getNextAddress it now sets an explicit gap policy.

I don't think this should be a problem, but wanted to call attention to this just in case it is.

Reply:
Yah that should be fine, maybe we can let the user toggle that in the future, but I believe explicit setting to WRAP by default is the way to go.  We've really only seen some exchanges and other use cases need to disregard the policy or error, otherwise I prefer the WRAP.